handlers/challenge_handler.py

```diff
@@ -338,10 +338,17 @@ def execute(self, slack_wrapper, args, channel_id, user_id):
         solver_list = [member['user']['name']]
 
         # Find additional members to add
-        for addSolve in additional_args:
-            if not addSolve in solver_list:
-                solver_list.append(addSolve)
-                additional_solver.append(addSolve)
+        for add_solve in additional_args:
+            if (add_solve.startswith("<@")) and (add_solve.endswith(">")):
+                # slack @notation (resolve to real username)
+                add_solve = add_solve[2:-1].upper()
+                user_obj = slack_wrapper.get_member(add_solve)
+                if (user_obj['ok']):
+                    add_solve = user_obj['user']['name']
+
+            if not add_solve in solver_list:                
```

Comment:
Preferred: `if add_solve not in solver_list`

Reply:
Though I truly cannot understand, how one would prefer this over parenthesis, which makes it directly obvious, which conditions belong together, without having to think a second about operator precedence (and thus improve code readability), it's done...

Reply:
Python makes efforts to make the english grammatical sense the proper sense. `x not in y` is an operator with the same semantics as `not (x in y)`. You're just used to a language that makes it confusing.